src/pipemode_op/test/testRecordReader/TestRecordIOReader.cpp

```diff
@@ -76,40 +76,6 @@ TEST_F(RecordIOReaderTest, InvalidMagicNumber) {
         std::runtime_error);
 }
 
-
```

Comment:
no replacement test?

Reply:
Yeah - this one just didn't make sense anymore. Before I was testing that we'd fail if the record has a continuation flag. Every 32bit number is now valid.